### ISSUE TITLE
Option to hide facebook container icon

### ISSUE THREAD

**Issue by ink404** [mozilla/contain-facebook#712]
**Is your feature request related to a problem? Please describe.**
The facebook container icon can be intrusive to the experience of the website. 
I appreciate that I'm now aware via a visual indication that this is a place where facebook is attempting to track me and that the plugin is working, but the visual impact is especially in the case attached. 

**Describe the solution you'd like**
An option to disable the facebook container icon for certain webpages.

**Describe alternatives you've considered**
na

**Additional context**
screenshot: https://ibb.co/zJ7yYTf

**Comment by plyoung** [comment]
Similar issue closed over 6 months ago. Can we please get an option to hide these icons now?
https://github.com/mozilla/contain-facebook/issues/387

**Comment by K-no-Uta** [comment]
Yes please! It's incredibly annoying. I'd really love to be able to:

a) poke that button and choose "hide" option
b) right click and choose "hide all buttons on this pahe"
c) the opposite of b), aka "show all buttons on this page"

It's not helpful if the icon is in th middle of pictures or videos!
